<p>Erreur lors du forçage d'une mise à jour de la base de données <strong>Modification de la base de données</strong>.{{ $plugin_name }}</strong> plugin dans <a
        href="/instances/{{$instance_id}}"><strong>{{ $instance_name }}</strong></a>. Message: {[ $exception_message }}</p>
